### PR TITLE
Slime Radio Potion

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Bluespace slime extracts now have a new chemical reaction with water, which create slime radio potions. When applied to a simple animal, that mob gains an internal radio.
/:cl:

Why: Realistically the only thing anyone cares about from xenobio are the simple animals, and ways you can interact with them. Being a simple animal unable to effectively communicate sucks. The bluespace slime is an end game one, but it's reactions are largely worthless. This PR addresses all three of those points.

If someone wants to sprite some sort of fancy bluespace radio organ and refluff this as a babel fish knockoff that'd be cool too.

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
can i remove the organ from the simplemob and then implant it in a human?

**Comment by Anonmare** [comment]
If it's an implant and not a cyber organ, it is technically removable via surgery and an empty implant case, but I do not believe simplemobs are candidates for surgery.

**Comment by SgtxRootbeer** [comment]
yeah, but you can gib it and collect the organ from the gibs, somehow, correct?

**Comment by Anonmare** [comment]
No, there are two different kinds of implants: Subdermal and cyberorgan

This filepath is the same as a subdermal implant, meaning it won't be recovered if the implantholder is gibbed.

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -825,6 +825,26 @@
...
 	qdel(src)
 
+/obj/item/slimepotion/slimeradio
+	name = "bluespace radio potion"
+	desc = "A strange chemical that grants those who ingest it the ability to broadcast and recieve subscape radio waves."
+
+/obj/item/slimepotion/slime/slimeradio/attack(mob/living/M, mob/user)
+	if(!ismob(M))
+		return
+	if(!isanimal(M)) //only works on animals that aren't player controlled
```
Comment (XDTM) [inline code/modules/research/xenobiology/xenobiology.dm]: aren't player controlled?

**Comment by MrDoomBringer** [comment]
bigman
@KorPhaeron @WJohn gave it a quick shot waddya'll think

**Comment by XDTM** [comment]
it's never gonna be visible though, unless you want to have a chip instead of a potion

**Comment by KorPhaeron** [comment]
@XDTM the action button is visible though

**Comment by Xhuis** [comment]
Reminds me of that one SCP about the wireless router that generates confused timber wolves around it every full moon.

**Pull request merged**